test/unit/test_imagefuncs.py

```diff
@@ -26,6 +26,22 @@ def test_jpeg_from_numpy():
     assert len(jpeg2) > len(jpeg3)
 
 
+def test_bytestream_from_filename():
+    images_streams = []
+    images_streams.append(bytestream_from_filename("test/assets/cat.jpeg"))
+    images_streams.append(bytestream_from_filename("test/assets/cat.png"))
+    images_streams.append(bytestream_from_filename("test/assets/cat.png", jpeg_quality=95))
+    for i in images_streams:
+        assert isinstance(i, ByteStreamWrapper)
+        image = Image.open(i)
+        assert image.mode == "RGB"
+
+    # pixel based test, verified the image is correct by eye, then got a pixel whose value to check against
+    png_bytestream = bytestream_from_filename("test/assets/cat.png", jpeg_quality=95)
+    png_image = Image.open(png_bytestream)
+    assert png_image.getpixel((200, 200)) == (215, 209, 197)
```

Comment:
You might want to get a pixel that has different height and width values, just so that we don't accidentally miss an issue of flipping dimensions.